src/locales/en.json

```diff
@@ -0,0 +1,96 @@
+{
+  "translation": {
+    "common": {
+      "unabledecode": "Unable to decode address",
+      "copied": "Copied",
+      "detail": "Detail",
+      "overview": "Overview",
+      "block": "Block",
```

Comment:
Why `block/block_height/transactions/transaction_fee` is in common module ? What is `common` ?